can/interfaces/vector/canlib.py

```diff
@@ -49,9 +52,50 @@
     LOG.warning("Could not import vxlapi: %s", exc)
 
 
+def deprecated_args_alias(**aliases):
+    def deco(f):
+        @functools.wraps(f)
+        def wrapper(*args, **kwargs):
+            rename_kwargs(f.__name__, kwargs, aliases)
+            return f(*args, **kwargs)
+
+        return wrapper
+
+    return deco
```

Comment:
Not that there is anything wrong with this (in fact it is really neat) but these helpers shouldn't be here.

Either it should be moved into `can.util` or perhaps consider adding a dependency that could do it?
https://github.com/tantale/deprecated or equivalent.